codebuild/python_37.yml

```diff
@@ -0,0 +1,18 @@
+version: 0.2
+
+env:
+  variables:
+    TOXENV: "py37-integ"
+    AWS_ENCRYPTION_SDK_PYTHON_INTEGRATION_TEST_AWS_KMS_KEY_ID: >-
+      arn:aws:kms:us-west-2:658956600833:key/b3537ef1-d8dc-4780-9f5a-55776cbb2f7f
+
+phases:
+  install:
+    runtime-versions:
+      python: latest
+  build:
+    commands:
+      - pyenv install 3.7.9
+      - pyenv local 3.7.9
```

Comment:
For consistency should we explicitly install python in each case? Otherwise any changes in supported runtimes could break us.

Reply:
3.7 and 3.5 are not installed by CodeBuild so they need to be manually installed. As discussed, this is done here and not everywhere because it is slow. Will add a comment.

Also, the specific versions tested is complicated. Testing every patch version is too extreme, verifying the major/minor versions is more important than the specific patch versions.

```
    # The specific versions are manually installed
    # because they are not installed
    # by default in CodeBuild containers.
    # `pyenv` does not have
    # a nice way to just install
    # the latest patch version.
    # I have selected the current latest patch
    # rather than try
    # and manage a one-liner or script.
    # Testing every minor version
    # is too extreme at this time.
    # The choice of versions should be reviewed.
```